graphql/escalation.go

```diff
@@ -148,6 +148,17 @@ func (h *Handler) escalationPolicyFields() g.Fields {
 		"name":        &g.Field{Type: g.String},
 		"description": &g.Field{Type: g.String},
 		"repeat":      &g.Field{Type: g.Int},
+		"is_user_favorite": &g.Field{
+			Type:        g.Boolean,
+			Description: "Indicates this esclation policy has been marked as a favorite by the user.",
+			Resolve: func(p g.ResolveParams) (interface{}, error) {
+				ep, err := getPolicy(p.Source)
+				if err != nil {
+					return nil, err
+				}
+				return ep.IsUserFavorite(), nil
+			},
+		},
```

Comment:
We can leave this out, the old `graphql` package is deprecated and no longer updated, it only still exists for compatibility.
```suggestion
```

Reply:
addressed, Thanks!